src/cli/upload.rs

```diff
@@ -54,7 +83,11 @@ impl UploadCommand {
 
         log::trace!("Encoding binary model");
         rbx_binary::to_writer(&mut buffer, tree.inner(), &encode_ids)?;
-        do_upload(buffer, self.asset_id, &cookie)
+        if use_open_cloud {
+            return do_upload_open_cloud(buffer, universe_id, self.asset_id, &api_key);
+        } else {
+            return do_upload(buffer, self.asset_id, &cookie);
+        }
```

Comment:
You don't need `return` in Rust, you can just remove the semicolons here. See above comment for how I think this can be integrated into the parameter reading code.